components/ui/buttons/thick.styl

```diff
@@ -0,0 +1,3 @@
+.Button
+  &--thick
```

Comment:
@dzucconi the thickness in question

Reply:
Either change this variant to be self-contained (not dependent on `size-lg`) or make this naming more specific (`border-heavy`)